.github/workflows/npm-release.yml

```
@@ -0,0 +1,63 @@
+name: npm-release
+
+on:  # yamllint disable-line rule:truthy
+  workflow_dispatch:
+    inputs:
+      releaseVersion:
+        type: choice
+        description: "Select version increment type (follows Semantic Versioning)"
+        required: true
+        options:
+          - patch
+          - minor
+          - major
+      app-path:
+        type: choice
+        description: "Select the node service to release"
+        required: true
+        options:
```

```suggestion
        default: refarch-frontend
        options:
```
If there's just one node service, may be a good idea to set this as default.